Xamarin.Forms.Platform.Android/Renderers/ListViewAdapter.cs

```diff
@@ -535,6 +505,53 @@ void SelectItem(object item)
 			Select(position, view);
 		}
 
+		void UpdateSeparatorVisibility(Cell cell, bool cellIsBeingReused, bool isHeader, bool nextCellIsHeader, ConditionalFocusLayout layout, out AView bline)
+		{
+			bline = null;
+			if (cellIsBeingReused)
```

Comment:
That will not the same.. We need to update also for group header and other stuff.